tests/phpunit/elementor/modules/kit-elements-defaults/data/test-controller.php

```diff
@@ -183,6 +183,27 @@ public function test_update_item__sanitizes_globals_and_dynamics() {
 		], $this->kit->get_json_meta( Module::META_KEY ) );
 	}
 
+	public function test_update_item__sanitizes_invalid_html_elements() {
+		// Arrange.
+		$this->act_as_admin();
+
+		// Act.
+		$response = $this->send_request( 'PUT', '/kit-elements-defaults/button', [
+			'settings' => [
+				'text' => '<script>alert("error")</script> Some text after',
```

Comment:
maybe also add some before text?

Reply:
?

Reply:
You add text after the script. maybe also add some text before the script